integration_tests/scheduled_cork_test.go

```diff
@@ -129,82 +131,88 @@ func (s *IntegrationTestSuite) TestScheduledCork() {
 		currentBlockHeight := status.SyncInfo.LatestBlockHeight
 		targetBlockHeight := currentBlockHeight + 15
 
-		s.T().Log("scheduling cork calls")
+		s.T().Logf("scheduling cork calls for height %d", targetBlockHeight)
+		blockHeightBytes := sdk.Uint64ToBigEndian(uint64(targetBlockHeight))
+		corkID := hex.EncodeToString(crypto.Keccak256Hash(
```

Comment:
It's a method on a `Cork` and since we aren't using the gRPC cork client we never explicitly construct one, so I extracted the process. I recall trying to replace the tx client with a protoc-generated cork client but it was giving me issues.